### DISCUSSION TITLE
Add some more functions to solve mini games.

### DISCUSSION THREAD

**Discussion by damoshack54** [masterking32/MasterHamsterKombatBot#305] [General]
After the new updates the puzzle game and Hexa Puzzle function disappeared, they are not solved by bots and keys and coins are not taken.
Could you fix and check? Thanks.

**Comment by tboy1337** [comment]
https://github.com/masterking32/MasterHamsterKombatBot/issues/268#issuecomment-2318866602

**Comment by tboy1337** [comment]
Duplicate of https://github.com/masterking32/MasterHamsterKombatBot/issues/268

**Comment by damoshack54** [comment]
They don't work, they don't fix. Has anyone tried if they fix? After yesterday's update, the two minigames are no longer fixed..

**Comment by tboy1337** [comment]
It's complicated, refer to https://github.com/masterking32/MasterHamsterKombatBot/issues/268 for more information.
